src/pykeen/training/slcwa.py

```diff
@@ -35,6 +36,7 @@ def __init__(
         model: Model,
         triples_factory: CoreTriplesFactory,
         optimizer: Optional[Optimizer] = None,
+        lr_scheduler: Optional[Union[_LRScheduler, ReduceLROnPlateau]] = None,
```

Comment:
Maybe you should save this union as a type in the `pykeen/lr_schedulrers/__init__.py` file and just call it `LRScheduler`